### PR TITLE
new 3.0 things

### PR DESCRIPTION
**Pull request by paichlear**

### CODE REVIEW

File: enemy-data/automatons/ruin-drake-skywatch.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+It is said to be a war machine left behind by a now-destroyed nation. It mimics the appearance of the beings known as Vishaps, with some functional enhancements. Such machines have an even greater ability to “absorb” — or perhaps the right word is “counteract” — elemental powers than the vishaps.  
+How, you wonder, did a civilization that could make such frightening machines of war collapse?  
+
+## Weakspot Details
+
+Cores.  
+
+## Energy drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/automatons/ruin-drake-skywatch.md:18]: 66%: common orb
33%: common orb
death: 2 common particles

File: enemy-data/bosses/overworld-bosses/jadeplume-terroshroom.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Its intelligence has already reached and perhaps has even surpassed real animals, and it will actively hunt prey to obtain sustenance. Its tail has a lovely jade plumage that seems very similar to that of the ancient sacred beast that the Spantamad school reveres. As far as the logic of mimesis is concerned, it seems quite likely that the two beings once walked the earth together.  
+
+## Weakspot Details
+
+Unknown.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/bosses/overworld-bosses/jadeplume-terroshroom.md:15]: 66%: dendro orb
33%: dendro orb
death: dendro orb + dendro particle

File: enemy-data/bosses/overworld-bosses/regisvines/regisvines.md [hunk 3]
```diff
@@ -14,14 +14,14 @@ Regisvine will only have a weakspot on their corolla after being stunned twice.
...
 | HP Threshold | Type | Element |
 | :--- | :--- | :--- |
-| 75% HP | 1 particle | own element   
-| 50% HP | 1 particle | own element  
-| 50% HP | 1 particle| own element 
-| Death | 1 orb | own element 
+| 75% HP | 1 Particle | Own Element |  
+| 50% HP | 1 Particle | Own Element | 
+| 50% HP | 1 Particle| Own Element |
+| Death | 1 Orb | Own Element |
```
Comment (ItzSomebody) [inline enemy-data/bosses/overworld-bosses/regisvines/regisvines.md:20]: 1 Orb, 1 Orb, 1 Orb, 1Orb

File: enemy-data/humans/eremites/eremite-axe-vanguard.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+These ax-wielding ladies often wade into the slaughter without fear, creating openings for their comrades to capitalize on.
+They will sometimes sit by the fireside in their sand-swept homeland and play songs that were lost in that golden sea.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-axe-vanguard.md:18]: 60%: common particle
death: common particle

File: enemy-data/humans/eremites/eremite-crossbow.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The Eremite tradition is to use bows to engage their foes, but ever since the Akademiya permitted them to buy crossbows at the bazaars, only the proudest shooters would still use bows, which take many years of practice to master.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-crossbow.md:17]: 60%: common particle
death: common particle

File: enemy-data/humans/eremites/eremite-crossbow.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+| Physical | -20% |
+| Pyro | 10% |
+| Dendro | 10% |
+| Hydro | 10% |
+| Electro | 10% |
+| Anemo | 10% |
+| Cryo | 10% |
+| Geo | 10% |
+
+## Stat Block
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-crossbow.md:32]: 60: 10630.731491043
65: 13976.21506251162
70: 17582.516413765494
75: 22000.76357410103
80: 28363.94175975211
85: 36237.78547225706
90: 44520.347078810446
95: 54514.39322714042
100: 66552.02486602496

File: enemy-data/humans/eremites/eremite-daythunder.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+## Eremite Daythunder
+
+A true scion of a warrior tribe, such warriors see battle as their ordained purpose, and unlike some barbarian who wields a large ax with ease, these tribespeople are also quite well-versed in the lore and songs of the desert ruins.  
+An ominous, fragmented spirit is sealed within the weapon they use, awakening in moments of peril. Once it does, it can only be silenced by the cries of the vanquished — whether that be its wielder, or its wielder’s foe.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-daythunder.md:16]: 75%: Electro particle
50%: Electro Particle
25%: Electro Particle
death: 2x Electro Particles

File: enemy-data/humans/eremites/eremite-daythunder.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+| Physical | -20% |
+| Pyro | 10% |
+| Dendro | 10% |
+| Hydro | 10% |
+| Electro | 10% |
+| Anemo | 10% |
+| Cryo | 10% |
+| Geo | 10% |
+
+## Stat Block
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-daythunder.md:33]: 60: 35435.76997828111
65: 46587.38136123866
70: 58608.38530278951
75: 73335.87514793873
80: 94546.46810737252
85: 120792.61258730292
90: 148401.14998362958
95: 181714.6355855316
100: 221840.07250379026

File: enemy-data/humans/eremites/eremite-desert-clearwater.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+A young lady lost in the ancient dream of the desert songs. To her, the thrill of bloody battle is her true reward. Legend has it that the women of this tribe are often afflicted by the wandering spirits of the desert, a condition that leads many of them to an early end. Perhaps this one can only be free of these apparitions when her blood is up in combat.  
+An ominous, fragmented spirit is sealed within the weapon she uses, awakening in moments of peril. Once it does, it can only be silenced by the cries of the vanquished — whether that be its wielder, or its wielder’s foe.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-desert-clearwater.md:18]: 75%, 50%, 25%: Hydro particle
death: 2x Hydro particles

File: enemy-data/humans/eremites/eremite-desert-clearwater.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+| Physical | -20% |
+| Pyro | 10% |
+| Dendro | 10% |
+| Hydro | 10% |
+| Electro | 10% |
+| Anemo | 10% |
+| Cryo | 10% |
+| Geo | 10% |
+
+## Stat Block
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-desert-clearwater.md:33]: 60: 31892.194473129
65: 41928.64518753486
70: 52747.54924129648
75: 66002.29072230309
80: 85091.82527925633
85: 108713.35641677119
90: 133561.04123643134
95: 163543.17968142126
100: 199656.0745980749

File: enemy-data/humans/eremites/eremite-linebreaker.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+This warrior wields a unique fist blade and joined the Eremites to seek out stronger foes. Rather than the pay, these warriors fight more to attain greater martial might.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-linebreaker.md:17]: 60% and death: common particle

File: enemy-data/humans/eremites/eremite-ravenbeak-halberdier.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+These warriors use Ravenbeak Halberds in battle. It is said that ascetics who served the deity of the desert would carry these halberds as symbols of their office. These weapons once had a true name, but it has long been lost to the desert sands.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-ravenbeak-halberdier.md:17]: 60% and death: common particle

File: enemy-data/humans/eremites/eremite-ravenbeak-halberdier.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+| Physical | -20% |
+| Pyro | 10% |
+| Dendro | 10% |
+| Hydro | 10% |
+| Electro | 10% |
+| Anemo | 10% |
+| Cryo | 10% |
+| Geo | 10% |
+
+## Stat Block
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-ravenbeak-halberdier.md:32]: 60: 10630.731491043
65: 13976.21506251162
70: 17582.516413765494
75: 22000.76357410103
80: 28363.94175975211
85: 36237.78547225706
90: 44520.347078810446
95: 54514.39322714042
100: 66552.02486602496

File: enemy-data/humans/eremites/eremite-sunfrost.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+A professional mercenary who cares about nothing outside of work and the mission at hand. To such people, their only game plan is to complete the mission, and their high asking prices on the market match that committment.  
+An ominous, fragmented spirit is sealed within the weapon they use, awakening in moments of peril. Once it does, it can only be silenced by the cries of the vanquished — whether that be its wielder, or its wielder’s foe.   
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-sunfrost.md:18]: 75%, 50%, 25%: cryo particle
death: 2x cryo particle

File: enemy-data/humans/eremites/eremite-sunfrost.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+| Physical | -20% |
+| Pyro | 10% |
+| Dendro | 10% |
+| Hydro | 10% |
+| Electro | 10% |
+| Anemo | 10% |
+| Cryo | 10% |
+| Geo | 10% |
+
+## Stat Block
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-sunfrost.md:33]: 60: 31892.194473129
65: 41928.64518753486
70: 52747.54924129648
75: 66002.29072230309
80: 85091.82527925633
85: 108713.35641677119
90: 133561.04123643134
95: 163543.17968142126
100: 199656.0745980749

File: enemy-data/humans/eremites/eremite-sword-dancer.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+This desert warrior wields a large curved blade in battle. The skills such fighters boast of have been passed down since the days when their now-lost civilization was still extant, and are not only techniques to be used in battle, but also serve well as a performing art popular in various lands.  
+It is said that this sword art was once practiced from horseback.  
+
+## Weakspot Details
+
+Head.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/humans/eremites/eremite-sword-dancer.md:18]: 60% and death: common particle

File: enemy-data/mystical-beasts/fungi.md [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+
+Research has shown that this sort of fungal beast’s internal electric current will change periodically. If further developed, perhaps this study may give rise to a wide range of uses…  
+
+## Weakspot Details
+
+Unknown.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/mystical-beasts/fungi.md:15]: death: common particle

File: enemy-data/mystical-beasts/shrooms.md [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+Going well past mere appearances, this being even imitates the habits of birds. It can use its spore-built wings to fly in the sky, spreading yet more spores to yet more distant places.
+This creature has a position of command amongst colonies of Fungi and exhibits heightened intelligence to match.
+
+## Weakspot Details
+
+Unknown.  
+
+## Energy Drops
+
+Unknown.
```
Comment (ItzSomebody) [inline enemy-data/mystical-beasts/shrooms.md:16]: 60% and death: own element particle

**Review by ItzSomebody — CHANGES_REQUESTED** [review]

**Review by ItzSomebody — APPROVED** [review]

**Review by ReensW — APPROVED** [review]

**Pull request merged**